src/app/shared/alert/alert.component.html

```diff
@@ -1,5 +1,7 @@
 <div *ngFor="let alert of alerts | async">
-  <ngb-alert [type]="alert.type" [dismissible]="alert.dismissible" (closed)="onAlertClosed(alert)" [ngClass]="{'mt-0 mb-0 rounded-0 w-100': isMain()}">
-    <div [ngClass]="{'container': isMain()}"><span [innerHTML]="alert.message"></span></div>
+  <ngb-alert [type]="alert.type" [dismissible]="alert.dismissible" (closed)="onAlertClosed(alert)" [ngClass]="{'mt-0 mb-0 rounded-0 w-100': isMain()}" role="alert" aria-live="assertive" [attr.aria-label]="alert.type === 'danger' ? 'Error alert' : alert.type === 'warning' ? 'Warning alert' : 'Information alert'">
+    <div [ngClass]="{'container': isMain()}" role="alertdialog" aria-describedby="alert-message">
+      <span id="alert-message" [innerHTML]="alert.message"></span>
```

Comment:
This must change.
The `alert-message` is not distinct and with the `*ngFor` above we will have non-distinct `id` attributes.

Reply:
Also, the `aria-describedby` should not be needed for when the content is nested inside.